configs/98netflix

```diff
@@ -0,0 +1,6 @@
+#!/system/bin/sh
```

Comment:
There's an empty kylepro device repository in CyanogenMod's GitHub as well: https://github.com/CyanogenMod/android_device_samsung_kylepro